client/gutenberg/extensions/simple-payments/edit.jsx

```diff
@@ -0,0 +1,473 @@
+/** @format */
```

Comment:
I'll go on the record to say that I'm not a fan of the `.jsx` extension.

I originally used `.js` intentionally. It was not an oversight.

I'd rather avoid the rename, but it's not a big deal or of high importance.

Some context: p1541419595035500-slack-calypso-framework